BrowserKit/Sources/Common/Theming/DefaultThemeManager.swift

```diff
@@ -21,15 +23,18 @@ public final class DefaultThemeManager: ThemeManager, Notifiable {
         }
 
         enum PrivateMode {
-            static let isOn = "profile.PrivateModeStatus"
+            static let byWindowUUID = "profile.PrivateModeWindowStatusByWindowUUID"
+
+            static let legacy_isOn = "profile.PrivateModeWindowStatus"
```

Comment:
Should this be "profile.PrivateModeStatus"?

Reply:
Yes, thank you for catching. Fixed